dist/dropdown/ds6/dropdown.css

```diff
@@ -0,0 +1,245 @@
+div.listbox__option[role^="option"]:active .menu__status,
+a.fake-menu__item:active .menu__status,
+button.fake-menu__item:active .menu__status,
+div.menu__item[role^="menuitem"]:active .listbox__status,
+div.listbox__option[role^="option"]:active .listbox__status,
+a.fake-menu__item:active .listbox__status,
+button.fake-menu__item:active .listbox__status,
+div.menu__item[role^="menuitem"]:active .fake-menu__status,
+div.listbox__option[role^="option"]:active .fake-menu__status,
+a.fake-menu__item:active .fake-menu__status,
+button.fake-menu__item:active .fake-menu__status {
+  color: #c7c7c7;
+}
+div.menu__item[role^="menuitem"]:not(:last-child),
+div.listbox__option[role^="option"]:not(:last-child),
+a.fake-menu__item:not(:last-child),
+button.fake-menu__item:not(:last-child) {
+  margin-bottom: 1px;
+}
+div.menu__item[role^="menuitem"][aria-checked="true"] .menu__status,
+div.listbox__option[role^="option"][aria-selected="true"] .menu__status,
+div.menu__item[role^="menuitem"][aria-checked="true"] .listbox__status,
+div.listbox__option[role^="option"][aria-selected="true"] .listbox__status {
+  visibility: visible;
+}
+a.fake-menu__item {
+  color: #111820;
+  text-decoration: none;
+}
+a.fake-menu__item:hover,
+a.fake-menu__item:focus {
+  text-decoration: underline;
+}
+a.fake-menu__item[aria-current="page"] .fake-menu__status {
+  visibility: visible;
+}
+button.fake-menu__item {
+  background-color: #ffffff;
+  border: 0;
+  color: #111820;
+  display: block;
+  text-align: left;
+}
+.menu__status,
+.listbox__status,
+.fake-menu__status {
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+  color: #111820;
+  fill: currentColor;
+  height: 1em;
+  stroke: currentColor;
+  stroke-width: 0;
+  visibility: hidden;
+  width: 1em;
+}
+span.menu__status,
+span.listbox__status,
+span.fake-menu__status {
+  background-image: url('data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20width%3D%278%27%20height%3D%277%27%3E%3Cpath%20fill%3D%22none%22%20stroke%3D%22%23111820%22%20d%3D%22M0%203.70913219%202.21049749%206%208%200%22%3E%3C/path%3E%3C/svg%3E');
+  background-position: right center;
+  background-repeat: no-repeat;
+  height: 1em;
+  -ms-flex-negative: 0;
+      flex-shrink: 0;
+  width: 1em;
+}
+.menu__items--grow[role="menu"],
+.menu__items--grow-reverse[role="menu"],
+.listbox__options--grow[role="listbox"],
+.listbox__options--grow-reverse[role="listbox"],
+.fake-menu__items--grow,
+.fake-menu__items--grow-reverse {
+  min-width: calc(100%);
+  width: auto;
+}
+.menu__items--grow-reverse[role="menu"],
+.listbox__options--grow-reverse[role="listbox"],
+.fake-menu__items--grow-reverse {
+  right: 0;
+}
+.listbox {
+  display: inline-block;
+  box-sizing: border-box;
+  width: 100%;
+}
+.listbox:hover > .listbox-btn > input.listbox-btn__control {
+  border-color: #4295ff;
+}
+.listbox > .listbox-btn > input.listbox-btn__control {
```

Comment:
Do you think there's worth in changing `.listbox-btn` from a BEM block to a BEM element? Then specificity is reduced in quite a few places.

For example:

`.listbox > .listbox-btn {`

Becomes just

`.listbox__control {` (or some other name)

Or do you think `listbox-btn` is usable elsewhere? Like we do with `.expand-btn`? If so, we'll keep it as is.

Reply:
What happens to the input?:
```
.listbox__control > input { ... }
```
With no class for the nested input?

Reply:
That works.

Reply:
Just confirming, `listbox-btn` goes away in favor of `listbox__control`.

Reply:
Okay. This does mean more changes to my component code. [sigh]

Small price for improvements. 😉